traefik/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: traefik
 description: A Traefik based Kubernetes ingress controller
 type: application
-version: 10.1.2
+version: 10.2.0
```

Comment:
```suggestion
version:  10.1.7
```

Reply:
Chart version updated

Reply:
My mistake. We are following semver, then: 

- enhancement: 1.X.0
- bug: 1.1.X
- breaking: x.0.0

The version you initially set was correct as it is an enhancement.